### PR TITLE
Action performed ready to review

### PR DESCRIPTION
**Pull request by Dheerajjnv**
What does this change do?
-Checking type of Ip configuration
-classify the quadrant based on the given set of point.

Any additional information for the reviewer to start
NA

How should this be manually tested?
We need window OS in which Rust is installed. then execute this rust program

Which Trello task and sub-task does this change relate to?

Are there any changes pending?
No
Does any team have to be notified of changes in this feature?
Yes
Definition of Done:

[x] Is there >90% unit test code coverage?
if not please mention the Code Coverage of code.
[ ] Does this PR add new dependencies? If so, please list out the same.
 Will this feature require a new piece of infrastructure to be implemented?
[ ] Is there appropriate logging included?
[x] Does the project compile ok?
Yes
[x] Have Clippy violations been fixed?
Yes
[x] Have Code is properly formatted?
Yes

### CODE REVIEW

**Comment by Dheerajjnv** [comment]
Cargo missed from parent

**Comment by Dheerajjnv** [comment]
Addig path for action

File: Assignments/assignment6/src/ip_configuration.rs [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+//ClassA, ClassB, ClassC, ClassD, InvalidConfiguration.
+#[derive(PartialEq, Eq, Debug)]
+pub enum IpAddress {
+    _ClassA(String),
+    _ClassB(String),
+    _ClassC(String),
+    _ClassD(String),
+    _InvalidConfiguration(String),
+}
+// This Function classify the ip_configuration.
```
Comment (ankit-mogha) [inline Assignments/assignment6/src/ip_configuration.rs:11]: use a good comment

Comment (ankit-mogha) [inline Assignments/assignment6/src/ip_configuration.rs]: in place of "this" use function name

File: Assignments/assignment6/src/quadrant.rs [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+            x_value, y_value
+        ),
+        _ => {
+            format!(
+                "Position::Fourth(Coordinate::Abscissa {}, Coordinate::Ordinate {} ",
+                x_value, y_value
+            )
+        }
+    }
+}
```
Comment (ankit-mogha) [inline Assignments/assignment6/src/quadrant.rs:41]: comments missing

File: Assignments/assignment6/src/test/quadrant_test.rs [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+#[test]
+fn fourth_quadrant_success() {
+    use crate::quadrant;
+    let points = quadrant::_Points { x: -1, y: -2 };
+    let test_output = quadrant::quadrant_evaluation(&points);
+    assert_eq!(
+        test_output,
+        "Position::Fourth(Coordinate::Abscissa -1, Coordinate::Ordinate -2 ".to_string()
+    );
+}
```
Comment (ankit-mogha) [inline Assignments/assignment6/src/test/quadrant_test.rs]: put all test cases under one test.rs file

Comment (Dheerajjnv) [reply Assignments/assignment6/src/test/quadrant_test.rs]: Pankaj told me to batter to do in a directory of test

File: Assignments/assignment6/src/quadrant.rs [hunk 4]
```diff
@@ -0,0 +1,29 @@
+pub struct _Points {
+    pub x: i32,
+    pub y: i32,
+}
+pub fn quadrant_evaluation(points: &_Points) -> String {
```
Comment (NitinSaxenait) [inline Assignments/assignment6/src/quadrant.rs:17]: comments missing in this file

File: Assignments/assignment6/src/test/ip_configuration_test.rs [hunk 5]
```diff
@@ -0,0 +1,52 @@
+#[cfg(test)]
+#[test]
+fn class_a_success() {
```
Comment (NitinSaxenait) [inline Assignments/assignment6/src/test/ip_configuration_test.rs]: use some comments in test file to divide functions based on their test

File: Assignments/assignment6/src/test/ip_configuration_test.rs [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+    use crate::ip_configuration;
```
Comment (NitinSaxenait) [inline Assignments/assignment6/src/test/ip_configuration_test.rs]: use single test.rs

**Review by NitinSaxenait — CHANGES_REQUESTED** [review]
1.Use single test file  [**Requesting change only for this reason]
2.Use some comments in test file also

**Review by NitinSaxenait — APPROVED** [review]
I have asked dheeraj to use single test.rs but he told me that he was told by Pankaj to use different file for test.
On the basis of this i am approving this change.

**Review by NitinSaxenait — APPROVED** [review]

**Review by ankit-mogha — APPROVED** [review]

**Pull request merged**